### ISSUE TITLE
prevent players from being completely invisible

### ISSUE THREAD

**Issue by HeladoDeBrownie** [HeladoDeBrownie/Nexus#83]
a player with only transparent pixels in their sprite cannot be detected by any current means

**Comment by HeladoDeBrownie** [comment]
related: players that are invisible because they're obstructed by other players

**Comment by HeladoDeBrownie** [comment]
perhaps add a small indicator over the head of other players

**Comment by HeladoDeBrownie** [comment]
resolved by https://github.com/HeladoDeBrownie/Nexus/commit/31eb806683a0dd6592e2d7204254c16ac89238af. the indicator can be tweaked further if necessary to make it less obtrusive when it's not needed
